client/.vscodeignore

```diff
@@ -1,9 +1,14 @@
-.vscode/**
-typings/**
-out/test/**
-test/**
-src/**
-**/*.map
-.gitignore
-tsconfig.json
-vsc-extension-quickstart.md
+.vscode/**
+typings/**
+out/test/**
+test/**
+src/**
+**/*.map
+.gitignore
+tsconfig.json
+vsc-extension-quickstart.md
+!node_modules/vscode-languageserver
+!node_modules/vscode-languageclient
+!node_modules/vscode-languageserver-protocol
```

Comment:
I'd move the `!…` lines to the end. Currently, it looks like you're reintroducing files _before_ even ignoring them via `**/node_modules/` :thinking: You could also probably keep those lines separated from the rest with a blank line.

```
foo/bar
**/plop/

!pala/poum
!plop/pouloulou
```

:woman_shrugging:

Reply:
OK!